lib/NeuralPDELogging/src/NeuralPDELogging.jl

```diff
@@ -21,4 +21,4 @@ function NeuralPDE.logscalar(logger::TBLogger, scalar::R, name::AbstractString,
     nothing
 end
 
-end
```

Comment:
This is the end for the module? The tests will fail with this change. Was this tested locally?